src/css/style.scss

```
@@ -1,558 +1,383 @@
 	}
 }
 
-/* Dashboard Styles */
-.dashboard-container {
-	background-color: #f7f7f7;
-	border: 1px solid #ddd;
-	border-radius: 8px;
-	margin: 0 auto;
-	max-width: 1000px;
-	padding: 20px;
+@keyframes contentFadeIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "contentFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)